ClangPowerTools/ClangPowerTools/Helpers/ReflectionManager.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Reflection;
+
+namespace ClangPowerTools.Helpers
+{
+  public static class ReflectionManager
+  {
+    public static object GetProperty<T>(object obj, string propertyName) => typeof(T).GetProperty(propertyName).GetValue(obj);
```

Comment:
Can you tell me what happens when the `propertyName` will not be present in the `T` type? The same question for the `SetProperty` method

Reply:
When you use typeof(T).GetProperty(propertyName), you assume that the property you're trying to access exists. Otherwise, it will crash. I'll add some comments there.

Reply:
I added null-checks in case the object does not have a property with the `propertyName` name.